index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Playfair+Display:wght@400;700;900&family=Poppins:wght@400;700;900&display=swap");
+    </style>
+    <style>
+      @import url("https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap");
+    </style>
+    <link rel="stylesheet" href="/resources/css/reset.css" />
+    <link rel="stylesheet" href="/resources/css/style.css" />
+    <title>Portfolio</title>
+  </head>
+  <body>
+    <main>
+      <header class="top-nav-burger">
+        <nav class="container burger">
+          <div class="top-corner-image">
+            <img
+              src="/resources/images/image_geometry_1.png"
+              alt="corner image"
+            />
+          </div>
```

Comment:
As the logo is not part of the navigation, it is better and recommendable to have it outside of the `nav` tag, but still inside the `header` tag.

- [ ] Kindly, place the logo outside the `nav` tag but still inside the `header` :sun_with_face: